mvc/views/tag-helpers/authoring.rst

```diff
@@ -1,8 +1,376 @@
-.. include:: /../common/stub-topic.txt
-
-|stub-icon| Authoring Tag Helpers
+Authoring Tag Helpers
 ===========================================
 
-.. include:: /../common/stub-notice.txt
+By `Rick Anderson`_
+
+    - `Getting started with Tag Helpers`_
```

Comment:
I'm checking with an editor but UC is old school and now considered incorrect. Only proper names (products) use UC.